Xamarin.Forms.Platform.iOS/Renderers/FormattedStringExtensions.cs

```diff
@@ -61,35 +61,54 @@ internal static NSAttributedString ToAttributed(this Span span, Element owner, C
 			if (text == null)
 				return null;
 
+			var range = new NSRange(0, text.Length);
```

Comment:
Unused?

Reply:
Guilty as charged for copying from my iOS LabelRenderer work, will fix this too, thanks.